src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/csv/CSVResultsExtractor.java

```diff
@@ -336,8 +336,16 @@ private String findFieldValue(String header, Map<String, Object> doc, boolean fl
 
     private String quoteValueIfRequired(final String input, final String separator) {
 
```

Comment:
please remove empty lines at start of methods (here and everywhere in PR)

Reply:
ok